content/en/dashboards/widgets/topology_map.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Topology Map Widget
+kind: documentation
+description: "Displays a map of a service to all of the services that call it, and all of the services that it calls."
+widget_type: "topologymap"
+aliases:
+    - /dashboards/widgets/service_map
+further_reading:
+- link: "/dashboards/graphing_json/"
+  tag: "Documentation"
+  text: "Building Dashboards using JSON"
+- link: "/tracing/services/services_map/"
+  tag: "Documentation"
+  text: "Service Map"
+---
+
+The Topology Map widget displays a visualization of data sources and their relationships to help understand how data flows through your architecture. 
+
+## Setup
+
+### Configuration
+
+1. Choose the data to graph:
+    * Service Map: The node in the center of the widget represents the mapped service. Services that call the mapped service are shown with arrows from the caller to the service. To learn more about the Service Map, reference the [Service Map feature of APM][1].
+
+2. Enter a title for your graph.
+
+## API
+
+This widget can be used with the **Dashboards API**. See the [Dashboards API documentation][2] for additional reference.
+
+The dedicated [widget JSON schema definition][3] for the service map widget is:
+
+{{< dashboards-widgets-api >}}
+
+## Further Reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+[1]: /tracing/services/services_map/
+[2]: /api/v1/dashboards/
```

Comment:
```suggestion
[2]: /api/latest/dashboards/
```
This link seems to be 404ing, even though it redirects when I try it on production 🤔 
Seeing as though it redirects to latest anyway, let's try that and see if it fixes it.